src/components/form/utils/Label.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Intent } from '@blueprintjs/core';
+import { FormGroup } from '@blueprintjs/core';
+import type { ReactNode } from 'react';
+
+interface LabelProps {
+  children: ReactNode;
+  error?: string;
+  required?: boolean;
+  label?: string;
+  intent?: Intent;
+  labelFor?: string;
+  onClick?: () => void;
+}
+
+export function Label(props: LabelProps) {
```

Comment:
Let's discuss together about the purpose of this component.

Reply:
This component is non-usefull i removed it